src/screens/Drawer/TaskDetailsScreen.tsx

```diff
@@ -29,6 +32,12 @@ export const TaskDetailsScreen = (props) => {
         dispatch(resetSwapConfig());
     };
 
```

Comment:
new functionality - when pressing on the "request swap" button, the client will ask server for possible swap candidates based on task, role, and church id, and then these will be stored into the swapReducer.